app/pb_kits/playbook/pb_selectable_card/_selectable_card.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import classnames from 'classnames'
+import { Icon } from '../'
+
+import type {
+  InputCallback
+} from "../types"
+
+import {
+  buildAriaProps,
+  buildDataProps,
+  buildCss,
+  noop,
+} from '../utilities/props'
+
+type Props = {
+  aria: Object,
+  checked: Boolean,
+  children?: Array<React.ReactChild>,
+  className?: String,
+  dark?: Boolean,
+  data: Object,
+  disabled?: Boolean,
+  icon?: Boolean,
+  id?: String,
+  multi?: Boolean,
+  name?: String,
+  onChange: InputCallback,
+  onSelect: InputCallback,
+  onUnselect: InputCallback,
+  text?: String,
+  value?: String
+};
+
+const SelectableCard = ({
+  aria = {},
+  checked = false,
+  children,
+  className,
+  dark = false,
+  data = {},
+  disabled = false,
+  icon = true,
+  id = null,
+  multi = true,
+  name,
+  onChange = noop,
+  onSelect = noop,
+  onUnselect = noop,
+  text,
+  value,
+  ...props
+}: Props) => {
+  const ariaProps = buildAriaProps(aria)
+  const dataProps = buildDataProps(data)
+  const handleChange = event => {
+    onChange(event)
+    event.target.checked ?
```

Comment:
What's the use case of having the `onSelect` and `onUnselect`? My understanding is that exposing the `onChange` would be enough for the client to figure out if it's selected or not.

Reply:
I have the same question.. It seems to not be used at all here.

Reply:
I'm currently not using the `onSelect` or `onUnselect`. Those were added because I think this kit was originally created by looking at the Toggle Kit. They can be removed. `OnChange` does everything I need.

Reply:
@christinaatai please remove `onSelect` and `onUnselect` in every place it exists and replace it with `onChange` in your docs.

example of how the callback function works:

```jsx
event => console.log(`The radio that changed has a name of ${event.target.name}, a value of ${event.target.value} and checked=${event.target.checked}`)
```